tests/resources/serializers/conftest.py

```diff
@@ -36,3 +36,573 @@ def enhanced_full_record(enhanced_full_record, parent_record):
     """Enhanced full record metadata with added parent metadata."""
     enhanced_full_record["parent"] = parent_record
     return enhanced_full_record
+
+
+# TODO: Use this representation in all serializer tests
+@pytest.fixture
+def full_record_to_dict():
+    """The to_dict() representation of a full record.
+
+    THIS is the representation that a serializer gets as input.
```

Comment:
The full_record / minimal_record in root conftest.py are (were) meant as record creation input from the REST API / service side of things, but they have been altered over time. Relying on them is more confusing/clunky than not, ergo the creation of clear to_dict() fixtures here.